ovos_utils/waiting_for_mycroft/base_skill.py

```diff
@@ -490,8 +434,7 @@ def voc_match(self, utt, voc_filename, lang=None, exact=False):
                                                  voc_filename + '.voc'))
 
             if not voc or not exists(voc):
-                raise FileNotFoundError(
-                    'Could not find {}.voc file'.format(voc_filename))
+                return False
```

Comment:
this was an unrelated fix not meant to be in this PR

if the voc file does not exist this now returns False instead of throwing an exception, if the file does not exist the voc doesnt match, no reason to fail the intent

log also removed because it can be very spammy